vision/unit_tests/module_in_frame/UnitTest.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
Let's add a unit tests to ensure that every circle is considered in the slope calculating, ie make an image of circles and ensure it has the expected slope values between circles. You will probably need to sort the lists before comparing them.